Projects/Online_Cloth_Strore/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Online Cloth Store</title>
+
+    <!-- font awesome cdn link  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+
+    <!-- custom css file link  -->
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+<body>
+<!-- Go to Top Button -->
+    <button class="topButton" id="topButton" onclick="goToTop();">Go to Top</button>
+    
+<!-- header section starts  -->
+
+<header>
+
+    <div class="header-1">
+
+        <a href="#" class="logo"><i class="fas fa-shopping-basket"></i>Primrose</a>
+
+        <form action="" class="search-box-container">
+            <input type="search" id="search-box" placeholder="search here...">
+            <label for="search-box" class="fas fa-search"></label>
+        </form>
+
+    </div>
+
+    <div class="header-2">
+
+        <div id="menu-bar" class="fas fa-bars"></div>
+
+         <nav class="navbar">
+            <a href="index.html">Home</a>
+            <a href="aboutus.html">About Us</a>
+            <a href="products.html">Products</a>
+            <a href="research.html">Research</a>
+            <a href="contact.html">Contact Us</a>
+           </nav>
+
+        <div class="icons">
+            <a href="#" class="fas fa-shopping-cart"></a>
+            <a href="#" class="fas fa-heart"></a>
+            <a href="#" class="fas fa-user-circle"></a>
+        </div>
+
+    </div>
+
+</header>
+
+
+
+<section class="home" id="home">
+
+    <div class="image">
+        <img src="images/home-img.png" alt="">
+    </div>
+
+    <div class="content">
+        <span>Affordable and High Quality Products</span>
+        <h3>Get the best deals</h3>
+        <p style="font-size: 2rem;color: #666;">Welcome to Primrose Clothing, where style meets comfort! Our online clothing store offers a wide range of fashionable and comfortable clothing for women who want to look and feel their best. From chic dresses to cozy sweaters, our collection is designed to fit your unique style and personality. We believe that fashion should not only look good, but also feel good, which is why we use only the highest quality materials for our clothing. With easy navigation, a user-friendly interface, and a wide variety of sizes and styles, shopping at Primrose Clothing is a breeze. Start browsing our collection today and discover your new favorite outfit!</p>
```

Comment:
It will be best if you use any Formatter  like prettier so that your code will be readable!
And Please don't use any static style tag in your HTML File;

Reply:
I have utilized a code formatter, such as Prettier, to ensure consistent and readable code. Additionally, all static style tags have been removed from the HTML file, and the styles are now organized in separate CSS files.